mantidproject/mantid
# IDR ET: link fold frames to TOSCA/TFXA

This is Trac ticket: http://trac.mantidproject.org/mantid/ticket/11017

To test:
- Open IDR > Energy Transfer
- Select either IRIS or OSIRIS
- Uncheck "Fold Multiple Frames"
- Select either TOSCA or TFXA
- "Fold Multiple Frames" should now be checked
- Check that documentation on the "Fold Multiple Option" makes reasonable sense

This does not disable the option for IRIS and OISRIS as it had typically been the default to have it enabled and if the data does not support the option then it is ignored, however you typically want this enabled for TOSCA, hence it being enabled by default when using this instrument.
